src/v2/view-builder/views/password/EnrollFactorPasswordView.js

```diff
@@ -4,19 +4,19 @@ import BaseForm from '../../internals/BaseForm';
 import BaseFactorView from '../shared/BaseFactorView';
```

Comment:
don't you need to copy `EnrollFactorPasswordView.js` into `EnrollAuthenticatorPasswordView.js` and make changes to it?

Reply:
I am probably missing context on why we need to copy. `EnrollFactorPasswordView` does all the work we need for Password Enroll.

Reply:
`EnrollFactorPasswordView` is to support enroll password for LEA. what if there are some selenium tests in monolith that verify this page? your change **might** breaking selenium tests. Sure, you can fix selenium test when upgrade widget in monolith. Wouldn't better we save this effort later? In other words, would be better that your changes are backward compatible.
Copy into `EnrollAuthenticatorPasswordView` (also update ViewFactor) would be just additive work and no harm.